rts/build/cmake/Util.cmake

```diff
@@ -257,6 +257,18 @@ macro    (catch_regex_group pattern group var str)
 endmacro (catch_regex_group)
 
 
+# macro that adds "fontconfig fontconfig-1" to find_library on win32
+macro(find_fontconfig_hack)
```

Comment:
I'm sorry for adding this, I don't know why the windows libraries are named weirdly in our dependencies